src/main/java/uk/gov/hmcts/reform/userprofileapi/infrastructure/clients/idam/IdamService.java

```diff
@@ -1,16 +1,17 @@
 package uk.gov.hmcts.reform.userprofileapi.infrastructure.clients.idam;
 
-import java.util.UUID;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.userprofileapi.domain.IdamRegistrationInfo;
 import uk.gov.hmcts.reform.userprofileapi.domain.service.IdentityManagerService;
 import uk.gov.hmcts.reform.userprofileapi.infrastructure.clients.CreateUserProfileData;
 
 @Component
 public class IdamService implements IdentityManagerService {
 
     @Override
-    public String registerUser(CreateUserProfileData requestData) {
+    public IdamRegistrationInfo registerUser(CreateUserProfileData requestData) {
         //TODO Currently Stubbed and not implemented yet
-        return UUID.randomUUID().toString();
+        return new IdamRegistrationInfo(HttpStatus.ACCEPTED);
```

Comment:
I can see you have work to do on this so probably irrelevant but wondered if this was the right level of abstraction for HTTP lingo

Reply:
Yes this is a todo (although sonar will probably complain about the TODO so will remove that).  

Was bit of an unknown so kept it simple for now.
The data model I agreed with Srini only required the http status code so that's the only thing I put in the class for now.  Is that what you meant?